bot/code_review_bot/workflow.py

```diff
@@ -95,8 +92,36 @@ def run(self, revision):
         issues, task_failures, notices, reviewers = self.find_issues(
             revision, settings.try_group_id
         )
-        if not issues and not task_failures and not notices:
-            logger.info("No issues or notices, stopping there.")
+
+        # Analyze issues in case the before/after feature is enabled
+        if revision.before_after_feature:
+            logger.info("Running the before/after feature")
+            # Search a base revision from the decision task
+            decision = self.queue_service.task(settings.try_group_id)
+            base_rev_changeset = (
+                decision.get("payload", {}).get("env", {}).get("GECKO_BASE_REV")
+            )
+            if not base_rev_changeset:
+                logger.info(
```

Comment:
Might be worth sending to sentry so we know how frequent this is?

Reply:
I never saw any task definition missing those values, I'm not sure this is even possible.
I removed that line so we are sure it raises the error.

Reply:
I discussed with Bastien, warnings are automatically raised as sentry errors (from tools module).